src/target/riscv/riscv.c

```diff
@@ -4803,21 +4838,21 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.name = "set_enable_eq_match_trigger",
 		.handler = riscv_set_enable_eq_match_trigger,
 		.mode = COMMAND_CONFIG,
```

Comment:
I think these commands should be `COMMAND_ANY`, not `COMMAND_CONFIG`.

Reply:
I think that we want this command to be `COMMAND_CONFIG` to avoid situations when, for example, we have already set some trigger in hw and that we disable it with this command.

Reply:
I don't see an issue here. The process of removing a trigger does not depend on trigger type. Moreover, if some time in the future it will depend on the type, it can still be discovered by reading the corresponding registers from the hart.

Reply:
I agree. Addresed.